config.txt

```diff
@@ -93,8 +93,15 @@ null,
  * pool_address	  - Pool address should be in the form "pool.supportxmr.com:3333". Only stratum pools are supported.
  * wallet_address - Your wallet, or pool login.
  * pool_password  - Can be empty in most cases or "x".
+ *
+ * Featured small pools       fee    
+ * pool.minemonero.pro:3333 - 0.9%
+ * pool.usxmrpool.com:3333  - 0.5%
+ * pool.xmr.pt:3333         - 0.4%
+ *
+ * We feature pools up to 1MH/s here. For a more complete list see M5M400's pool list at www.moneropools.com
  */
-"pool_address" : "pool.supportxmr.com:3333",
+"pool_address" : "pool.usxmrpool.com:3333",

```

Comment:
Hi! Pool owner here.

I'll make things clearer on the website soon, but we do not have automatic hopping detection. I do my best to block NiceHash's IPs. If I (we) catch someone sending, for example, 300 KH/s for 50 minutes than running away, I do my best to contact the person, and if I get no answer (or person doesn't want to talk), I block that miner's wallet and IP addresses. In two cases only, I have decided to not send the funds to these miners as they were CLEARLY abusing the pool.

And to make things very clear: there is absolutely no problem in connecting to the pool 10 or 20 hours a day. Many miners do so as it's simply a hobby for them (like gamers who stop mining while gaming). PPLNS is a "solution" but it penalizes smaller hobbyist miners, and I'm not sure I want to implement it.

PS: English is my second language. Please be indulgent. :-)